### ISSUE TITLE
Hide search filters if not functional

### ISSUE THREAD

**Issue by watchoutfreedom** [helpbuttons/helpbuttons#74]
Are we going to finish  search filters view? Would be good to enable search filters but maybe not needed the view for now. The field on top would be search place and the rest of the filters can be enabled or directly hidden if they''re not going to work for now.

**Comment by albjeremias** [comment]
good point, didn't work on that at all. i would prioritize the feeds of the button before diving into this. for now can I disable it?

**Comment by watchoutfreedom** [comment]
yes we can disable except for the search field I'd say
